### PR TITLE
Add .npmignore back to root

### PR DESCRIPTION
**Pull request by styfle**
### Description

The `.npmignore` file was removed in v1.3.0: [6885af6b0a6b984a90fe203f00451316656f1355](https://github.com/milligram/milligram/commit/6885af6b0a6b984a90fe203f00451316656f1355)

This PR adds it back so that the npm publish size is smaller.

### Code sample

You can see the effect of removing npmignore with [Package Phobia](https://packagephobia.now.sh/result?p=milligram@1.3.0)

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+.npmignore
+.sasslintrc
+.travis.yml
+backstop.conf.js
+bower.json
+changelog.md
+composer.json
+package.js
+package.json
+src
```
Comment (cjpatoilo) [inline .npmignore]: @styfle which your opinion about remove the source code of the request package?

Comment (styfle) [reply .npmignore]: You don’t need src, only dist

Comment (styfle) [reply .npmignore]: This is the same file from the previous commit. I’m fixing the regression.

**Comment by cjpatoilo** [comment]
@styfle Good call 🚀

**Pull request merged**